dotnet/src/SemanticKernel/Planning/Planners/SequentialPlanner.cs

```diff
@@ -11,6 +11,11 @@ namespace Microsoft.SemanticKernel.Planning.Planners;
 /// </summary>
 public class SequentialPlanner
 {
+    /// <summary>
+    /// Kernel associated with this planner.
+    /// </summary>
+    public IKernel Kernel { get; }
```

Comment:
why add this public property?

Reply:
Yeah, shouldn't be needed.

Reply:
If the expected pattern is to use the same kernel that was used to create the planner, I would really want access to the kernel expected to run the plan from the planner.

Reply:
Offline discussion - removing.